versions/development/SPEC.md

```diff
@@ -263,6 +264,49 @@ $float = (([0-9]+)?\.([0-9]+)|[0-9]+\.|[0-9]+)([eE][-+]?[0-9]+)?
 |`\'`|single quote|`\x22`|
 |`\"`|double quote|`\x27`|
 
+### Comments
+
+Comments are a useful way of providing helpful information such as workflow usage, requirements, copyright etc directly within the wdl file. Comments can be added anywhere within the `WDL` and are used to indicate text which should be ignored by an engine implementation. The one caveat to be aware of, is that within the `command` section, *ALL* text will be included in the underlying command and any lies prepended by `#` will not be ignored.
+
+A comment is started with the number sign `#`. Any text following the number sign will be completely ignored, regardless of its content by an engine. Comments can be placed at the start of a new line or after any declarations within the WDL itself. At the moment, there is no planned support for multi-line comments, instead simply use a `#` at the start of each line.
```

Comment:
Suggestion:
"there is no planned support for multi-line comments" => "there is no special syntax for multi-line comments"